tests/apollo/test_skvbc_preexecution.py

```diff
@@ -100,11 +99,12 @@ async def test_sequential_pre_process_requests(self, bft_network):
         bft_network.start_all_replicas()
         skvbc = kvbc.SimpleKVBCProtocol(bft_network)
 
+        await trio.sleep(5)
```

Comment:
My only concern with this approach is the entire test will be delayed by a total of 5x3x3 = 45sec